Makefile

```diff
@@ -0,0 +1,9 @@
+NAME	:= gotestsum
+SRC_EXT	:= gz
+VERSION	:= 0.4.0
+
+include packaging/Makefile_packaging.mk
+
+$(VERSION).tar.gz:
+	rm -f ./$@
+	curl -f -L -O 'https://github.com/gotestyourself/$(NAME)/archive/v$(VERSION).tar.gz'
```

Comment:
There should be rules to download source artefacts in `packaging/Makefile_packaging.mk` already:

https://github.com/daos-stack/gotestsum/blob/9643e6fb35ea164332364c901120e26ee97ad62c/packaging/Makefile_packaging.mk#L127-L141

and in particular it looks like this one:

https://github.com/daos-stack/gotestsum/blob/9643e6fb35ea164332364c901120e26ee97ad62c/packaging/Makefile_packaging.mk#L135-L137

should be doing it.

The recipe here is a bit interesting though.  Your _target_ is `$(VERSION).tar.gz` yet what's being downloaded is `v$(VERSION).tar.gz`.  Typo on the recipe target perhaps?